dev/flutter/todo/submissions/Dishal Kanishka/todo_application/lib/screens/view_todo.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+import 'package:todo_application/Class/todo.dart';
+import 'package:todo_application/Constants/colors.dart';
+
+class TodoView extends StatefulWidget {
+  Todo todo;
+  TodoView({Key? key, required this.todo}) : super(key: key);
+
+  @override
+  _TodoViewState createState() => _TodoViewState(todo: this.todo);
+}
+
+class _TodoViewState extends State<TodoView> {
+  Todo todo;
+  _TodoViewState({required this.todo});
```

Comment:
We could just access this with `widget.todo` from the state component below instead of this